CMakeLists.txt

```diff
@@ -206,6 +206,8 @@ add_library(lava.resource STATIC
         ${LIBLAVA_DIR}/resource/image.hpp
         ${LIBLAVA_DIR}/resource/mesh.cpp
         ${LIBLAVA_DIR}/resource/mesh.hpp
+        ${LIBLAVA_DIR}/resource/mesh.tpp
```

Comment:
If you're using a `.tpp` extension, you'll have to adapt the header install step to include those file types:
https://github.com/liblava/liblava/blob/5eb577536a3fc40cc4da48c567e97a3d61f474dd/CMakeLists.txt#L446

Reply:
In fact, I want to avoid introducing another extension, so I would move that part to a header file (like generic_mesh.hpp)